### PR TITLE
Release/0.10.0

### PR DESCRIPTION
**Pull request by garmoshka-mo**
.md-fab - fix bug (duplicate scss declaration) and 
change usage of color palette to more pastel (though this can be rejected)

### CODE REVIEW

**Comment by garmoshka-mo** [comment]
I signed it!

**Comment by EladBezalel** [comment]
I think this is not a valid PR

**Comment by garmoshka-mo** [comment]
> I think this is not a valid PR

Pretty constructive :)

**Comment by DevVersion** [comment]
@garmoshka-mo I think Elad just thought your PR isn't valid because you have about eight commits with the same commit message. In addition to that, you have some case-specific changes, but I will review it for your.

So in general, all PR's should contain squashed commits (into 1 single commit) + they should follow the contributing rules. ([See the rules](https://github.com/angular/material/blob/master/CONTRIBUTING.md))

File: gulp/config.js [hunk 1]
```diff
@@ -35,7 +35,7 @@ module.exports = {
     'src/core/style/layout.scss'
   ],
   paths: 'src/{components,services}/**',
-  outputDir: 'dist/',
+  outputDir: '../spoti/tags/app/assets/lib/',
```
Comment (DevVersion) [inline gulp/config.js]: This seems to be definitely wrong? `./dist` is our output directory.

File: gulp/tasks/build-scss.js [hunk 2]
```diff
@@ -33,12 +33,12 @@ exports.task = function() {
           .pipe(rename({ basename: filename }))
           .pipe(util.autoprefix())
           .pipe(insert.prepend(config.banner))
-          .pipe(gulp.dest(dest))
+          //.pipe(gulp.dest(dest))
```
Comment (DevVersion) [inline gulp/tasks/build-scss.js]: Why are you removing the destination / output from the gulp pipeline?

File: gulp/tasks/build-scss.js [hunk 3]
```diff
@@ -33,12 +33,12 @@ exports.task = function() {
...
           .pipe(gulpif(!IS_DEV, minifyCss()))
           .pipe(rename({extname: '.min.css'}))
           .pipe(gulp.dest(dest))
   );
-  streams.push(
+/*  streams.push(
       gulp.src(config.scssStandaloneFiles)
```
Comment (DevVersion) [inline gulp/tasks/build-scss.js]: Plus here - Why are you commenting the most things of the SCSS gulp task?

**Comment by EladBezalel** [comment]
@DevVersion i closed it because it's on:
- release branch
- commented out lines of code
- `../spoti/tags/app/assets/lib` ???
- 0.10.0 is old

and so on..

**Comment by DevVersion** [comment]
@EladBezalel Yeah, completely understand you :), you told him, why you closed it, because he requested constructive feedback.

**Comment by garmoshka-mo** [comment]
Ah, sorry guys, now i see. All the commits with "1" was made for my personal use.
I forgot that i made pull request and made personal commits to the same branch.
Created other PR - https://github.com/angular/material/pull/7539 - here is what was suggested changes

**Pull request closed without merging**